### PR TITLE
Feature/implement scroll

### PR DESCRIPTION
**Pull request by julia-thea**
- Implemented vertical scroll for the review list

- Implemented partial stars

@acdavitt @ChrisRPeterson @dylanreid7

### CODE REVIEW

File: client/components/RatingsAndReviews/RatingsAndReviews.jsx [hunk 1]
```diff
@@ -8,14 +8,18 @@ import { makeStyles } from '@material-ui/core/styles';
...
+    marginBottom: 35
   }
 }));
 
 const RatingsAndReviews = (props) => {
   const classes = useStyles();
   return (
     <>
-    <Typography variant="subtitle1">RATINGS & REVIEWS</Typography>
+    <Typography variant="subtitle1" class={classes.header}>RATINGS & REVIEWS</Typography>
```
Comment (acdavitt) [inline client/components/RatingsAndReviews/RatingsAndReviews.jsx:21]: Getting an error regarding line 21: class I think should be className?  It doesn't appear to be affecting the functionality, but just to get rid of the warning.

**Review by acdavitt — CHANGES_REQUESTED** [review]
Looks good!  Just one quick change requested regarding class -> className to remove a warning that is popping up.

File: client/components/RatingsAndReviews/RatingsAndReviews.jsx (hunk 1 above)
Comment (julia-thea) [reply client/components/RatingsAndReviews/RatingsAndReviews.jsx]: Addressed!

Comment (acdavitt) [reply client/components/RatingsAndReviews/RatingsAndReviews.jsx]: Thanks!

**Review by acdavitt — APPROVED** [review]

**Pull request merged**